README.md

```diff
@@ -23,7 +23,7 @@ decided for the approach with less maintenance effort, since it will automatical
 ### Running SILO locally with CLion
```

Comment:
I think it would be nice do document the temp folder somewhere. I'm not sure if this should be here, or in a separate documentation. We could also add a ticket for that.